### PR TITLE
DOCS-13049-verified-branch

### PR DESCRIPTION
**Pull request by IsaacEldridge**
commit d1239c2e7982edbe58a622a161c3f9e5bc0ab833
Merge: 4db24c6 f2aa154
Author: Isaac Eldridge <ieldridge@mulesoft.com>
Date:   Wed Jan 26 11:53:26 2022 -0800

    Merge branch 'v1.11' into DOCS-13049-add-rtfctl-commands-to-uninstall

commit 4db24c630c763d55f3be1bef3633c42e06421990
Merge: c35a258 a850fb7
Author: Isaac Eldridge <ieldridge@mulesoft.com>
Date:   Wed Jan 19 10:00:07 2022 -0800

    Merge branch 'v1.11' into DOCS-13049-add-rtfctl-commands-to-uninstall

commit c35a258be3bf8d3ab10103f85d7342c5fc41cf2f
Author: IsaacEldridge <ieldridge@mulesoft.com>
Date:   Thu Jan 13 10:05:41 2022 -0800

    revision

### CODE REVIEW

File: modules/ROOT/pages/uninstall-manual.adoc [hunk 1]
```diff
@@ -51,4 +55,12 @@ sudo umount /var/lib/gravity
 
 sudo rm -rf /opt/anypoint/runtimefabric /root/.gravity /rtf-user/.gravity /var/lib/gravity /tmp/gravity /tmp/assets393106389 /tmp/gravity-wizard167466702 
 ----
-Remove lines in the `/etc/fstab` file which are appended with the comment `# Added by RTF`. 
+
+=== Uninstall Runtime Fabric Without Deleting the Gravity Cluster
```
Comment (sathya0) [inline modules/ROOT/pages/uninstall-manual.adoc]:
```suggestion
== Uninstall Runtime Fabric Without Deleting the Gravity Cluster
```

File: modules/ROOT/pages/uninstall-manual.adoc [hunk 2]
```diff
@@ -51,4 +55,12 @@ sudo umount /var/lib/gravity
...
+
+You can uninstall Runtime Fabric but keep the Gravity cluster. This is useful in scenarios in which you want to reinstall Runtime Fabric in the same cluster. To do so, run the following commands on one of the controller nodes:
```
Comment (sathya0) [inline modules/ROOT/pages/uninstall-manual.adoc:61]:
```suggestion
You can uninstall Runtime Fabric but keep the gravity cluster. This is useful in scenarios in which you want to reinstall Runtime Fabric in the same cluster. To do so, run the following commands on one of the controller nodes:
```

File: modules/ROOT/pages/uninstall-manual.adoc [hunk 3]
```diff
@@ -3,16 +3,22 @@ ifndef::env-site,env-github[]
...
+
+Perform this task to uninstall Runtime Fabric and completely remove the gravity cluster from your VMs. 
+
+Alternatively, you can perform the first two steps (delete all applications and the Runtime Fabric in Runtime Manager), and 
+skip the remaining steps by re-imaging the VM disks.
+
+[IMPORTANT]
+To successfully complete this task, keep all nodes running and joined to the cluster. 
+
+. Using Runtime Manager delete all Mule applications running on Runtime Fabric.
```
Comment (sathya0) [inline modules/ROOT/pages/uninstall-manual.adoc]:
```suggestion
. Using Runtime Manager, delete all Mule applications running on Runtime Fabric.
```

**Review by sathya0 — DISMISSED** [review]

File: modules/ROOT/pages/uninstall-manual.adoc (hunk 2 above)
Comment (IsaacEldridge) [reply modules/ROOT/pages/uninstall-manual.adoc:61]: this is a proper noun

**Review by sathya0 — APPROVED** [review]

**Pull request merged**